portfolio/src/main/webapp/style.css

```diff
@@ -2,8 +2,54 @@
   margin-left: auto;
   margin-right: auto;
   width: 650px;
+  margin-top: 30;
```

Comment:
Alphabetize declarations in this file: https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order